packages/dataviews/src/components/dataviews-filters/index.tsx

```diff
@@ -226,8 +221,8 @@ function Filters() {
 		<HStack
 			justify="flex-start"
 			style={ { width: 'fit-content' } }
-			className="dataviews-filters__container"
```

Comment:
Removed the default `dataviews-filters__container` class, as it imposes layout styles that may not work well in free composition mode.

The component now accepts a `className` prop, allowing consumers to control its layout and appearance as needed.